pennylane/qnode_new/qubit_parameter_shift.py

```diff
@@ -0,0 +1,368 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.

```

Comment:
Would it be better maybe to name the file as `qubit_qnode.py` and elaborate on the content in docstring and documentation? I believe that that could make the association between the class and the file easier

Reply:
Yes, you're right - the name was chosen in case we have new analytic gradient classes in the future that don't use the parameter shift, but makes sense to leave that for now.